test/e2e/placement_selecting_resources_test.go

```diff
@@ -702,6 +702,14 @@ var _ = Describe("validating CRP when failed to apply resources", Ordered, func(
 		createWorkResources()
 
 		ns := appNamespace()
+		ns.SetOwnerReferences([]metav1.OwnerReference{
+			{
+				APIVersion: "another-api-version",
+				Kind:       "another-kind",
+				Name:       "another-owner",
+				UID:        "another-uid",
+			},
+		})
```

Comment:
why put this?

Reply:
This test is to create a resource in the member first and then create CRP. Without the fix, the apply will fail. It validates the CRP status.

For now, only when the ns is owned by other applier, CRP will fail.